AutomaticInterface/Tests/GeneratorTests.cs

```diff
@@ -34,13 +34,6 @@ private static string GenerateCode(string code)
             .Should()
             .BeEmpty();
 
-        sourceDiagnostics
-            .Where(d => d.Severity == DiagnosticSeverity.Error)
-            .Where(x => x.Id != "CS0246")
-            .Where(x => !MissingUsingsAreOk(x))
-            .Should()
-            .BeEmpty();
```

Comment:
This check was already covered in the broader check on lines 31-35:

https://github.com/codecentric/net_automatic_interface/blob/e151b0564933ca97e5685f4d93648adab5ce1ccb/AutomaticInterface/Tests/GeneratorTests.cs#L31-L35